### PR TITLE
Various changes

### PR DESCRIPTION
**Pull request by dvandyk**
- use a WET specific renormalization scale for ``UbLnuL`` operators, where ``U=u,c`` and ``L`` a charged lepton
 - make q2-integrated CP averaged and asymmetries in ``B->Vlnu`` decays cacheable
 - remove more unused code related to Minuit2 and PMC
 - update .gitignore
 - don't force developers to compile with ``-flto``, which is slow

and further smaller changes

### CODE REVIEW

**Review by mreboud — APPROVED** [review]
Two minor comments, apart from the current failure in the checks the rest looks good!

File: .github/workflows/macos-build+check.yaml [hunk 1]
```diff
@@ -66,7 +63,6 @@ jobs:
                 popd
                 pushd _build
                 ../_src/configure \
-                    --disable-pmc \
```
[the remark this answers is not in the document]
Comment (dvandyk) [reply .github/workflows/macos-build+check.yaml:69]: It should. I overlooked this one. ``configure`` ignores unknown options, interestingly.

File: eos/constraints/B-to-D-pi-l-nu.yaml [hunk 2]
```diff
@@ -610,282 +610,940 @@ B^0->(Dpi)^+l^-nu::KinematicalDistribution[w]@Belle-2018A:
 # Pheno analysis by BBGJvD of Belle data #
 ##########################################
 #
-# 2020
+# 2021
 #
 # {{{
-B^0->(Dpi)^+e^-nubar::NormalizedObservables@BBGJvD:2020A:
```
[the remark this answers is not in the document]
Comment (dvandyk) [reply eos/constraints/B-to-D-pi-l-nu.yaml:616]: Will do.

File: .github/workflows/macos-build+check.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (dvandyk) [reply .github/workflows/macos-build+check.yaml:69]: Sorry, I did exactly that here: remove the use of the (ignored) option

[the remark this answers is not in the document]
Comment (mreboud) [reply .github/workflows/macos-build+check.yaml:69]: Ah yes, sorry!

**Pull request merged**